components/add_groups_to_team_modal/add_groups_to_team_modal.tsx

```diff
@@ -109,11 +128,7 @@ export default class AddGroupsToTeamModal extends React.PureComponent {
         });
     }
 
-    handleSubmit = async (e) => {
-        if (e) {
-            e.preventDefault();
```

Comment:
`preventDefault` should still be called, but the if condition is not necessary with typescript anymore:

`e?.preventDefault?.();` does the trick

Reply:
Can you look again? This is probably outdated code.
It seems that `handleSubmit` no longer receives event.  `handleSubmit` is passed to multiselect component, which invokes `handleSubmit` with `Value[]` as argument, not `SyntheticEvent`.
https://github.com/mattermost/mattermost-webapp/blob/3d6ae3cbeff66663b28798e8c04fa22d0e5d6ab4/components/add_groups_to_team_modal/add_groups_to_team_modal.jsx#L289
https://github.com/mattermost/mattermost-webapp/blob/3d6ae3cbeff66663b28798e8c04fa22d0e5d6ab4/components/multiselect/multiselect.tsx#L36
https://github.com/mattermost/mattermost-webapp/blob/3d6ae3cbeff66663b28798e8c04fa22d0e5d6ab4/components/multiselect/multiselect.tsx#L177
https://github.com/mattermost/mattermost-webapp/blob/3d6ae3cbeff66663b28798e8c04fa22d0e5d6ab4/components/multiselect/multiselect.tsx#L220
https://github.com/mattermost/mattermost-webapp/blob/3d6ae3cbeff66663b28798e8c04fa22d0e5d6ab4/components/multiselect/multiselect.tsx#L229-L230

Reply:
You are absolutely right. It is kind of an anti-pattern though (I think) 🙇🏼‍♂️